### PR TITLE
Fix labelSelector query param inconsistency in docs

### LINKED ISSUE #10229: Label selector query parameter is labelSelector instead of label-selector?
Both api.md and labels.md says the query parameter is `label-selector`
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api.md
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/labels.md

But it seems like it should be `labelSelector`
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/api/swagger-spec/v1beta3.json
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/api/swagger-spec/v1.json

### PR DESCRIPTION
**Pull request by j3ffml**
Closes #10229. cc @bgrant0607

### CODE REVIEW

File: docs/labels.md [hunk 1]
```diff
@@ -82,8 +82,8 @@ _Set-based_ requirements can be mixed with _equality-based_ requirements. For ex
 
 LIST and WATCH operations may specify label selectors to filter the sets of objects returned using a query parameter. Both requirements are permitted:
 
-   * _equality-based_ requirements: `?label-selector=key1%3Dvalue1,key2%3Dvalue2`
-   * _set-based_ requirements: `?label-selector=key+in+%28value1%2Cvalue2%29%2Ckey2+notin+%28value3`
+   * _equality-based_ requirements: `?labelSelector=key1%3Dvalue1,key2%3Dvalue2`
+   * _set-based_ requirements: `?labelSelector=key+in+%28value1%2Cvalue2%29%2Ckey2+notin+%28value3`
```
Comment (lavalamp) [inline docs/labels.md]: This is missing a closing %29 ')'

File: docs/api.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ Some important differences between v1beta1/2 and v1beta3:
 * The namespace is required (for all namespaced resources) and has moved from a URL parameter to the path: `/api/v1beta3/namespaces/{namespace}/{resource_collection}/{resource_name}`. If you were not using a namespace before, use `default` here.
 * The names of all resource collections are now lower cased - instead of `replicationControllers`, use `replicationcontrollers`.
 * To watch for changes to a resource, open an HTTP or Websocket connection to the collection query and provide the `?watch=true` query parameter along with the desired `resourceVersion` parameter to watch from.
-* The `labels` query parameter has been renamed to `label-selector`.
+* The `labels` query parameter has been renamed to `labelSelector`.
```
Comment (lavalamp) [inline docs/api.md]: Did fieldSelector used to be field-selector, too?

Comment (lavalamp) [inline docs/api.md]: And was this a v1beta1->v1beta3 change, or a v1beta3->v1 change?

Comment (nikhiljindal) [inline docs/api.md]: Yes. It was beta1->3 change: https://github.com/GoogleCloudPlatform/kubernetes/blob/de457c3e493234a819294bd6ec1f18101b9c4242/pkg/api/unversioned.go#L52

Comment (jlowdermilk) [inline docs/api.md]: Added `fields -> fieldSelector` bulletpoint.

File: docs/labels.md (hunk 1 above)
Comment (jlowdermilk) [inline docs/labels.md]: Done.

**Comment by jlowdermilk** [comment]
@k8s-bot retest this please

**Comment by jlowdermilk** [comment]
FYI for @ixdy, k8s-bot seems confused; 0 failed tests == build/test failed?

For reviewers PTAL, this should be good to merge.

**Comment by lavalamp** [comment]
e2e failed because the test timed out while it was taking down the test cluster.

**Comment by lavalamp** [comment]
Clearly this doc change could not have caused that, though. LGTM

**Comment by mbforbes** [comment]
@lavalamp note for clueless oncalls like me: please add the `ok-to-merge` label as we're no longer scanning for `LGTM`s in order to merge, given the stricter policy. Thank you!

**Pull request merged**